### ISSUE TITLE
Parallel workers and merging of results

### ISSUE THREAD

**Issue by Conradowatz** [tls-attacker/Anvil-Web#14]
One usecase of the workers would be to split test execution upon different devices.
Example: to test a tls server completly you need 400 tests. You split them to 4 workers, each testing a copy of the same library. Every worker tests just 100 tests, but all 4 are running in parallel. They report their results to Anvil Web where the results are merged into one report.
This way test execution times can be reduced.
